reg/drone/service/actuator/esc/FaultFlags.0.1.uavcan

```diff
@@ -0,0 +1,33 @@
+# A collection of detailed fault flags indicating problems detected by the service provider.
+# A fault flag is set when the corresponding parameter exceeds its safe operating area (SOA).
```

Comment:
We need to specify where that SOA is defined. Do we specify it in the spec? Or is a per-vendor SOA?

Reply:
The term SOA is used in the general sense: https://en.wikipedia.org/wiki/Safe_operating_area
So it is up to the vendor to define it.

Reply:
Extended the docs to make this clear.